packages/stark-core/src/logging/services/logging.service.ts

```diff
@@ -77,12 +78,10 @@ export class StarkLoggingServiceImpl implements StarkLoggingService {
 			this.logUrl = this.backend.url + "/" + this.appConfig.loggingFlushResourceName;
 			this.generateNewCorrelationId();
 
-			this.store
-				.select<StarkLogging>((state: StarkLoggingApplicationState) => state.starkLogging)
-				.subscribe((starkLogging: StarkLogging) => {
-					this.starkLogging = starkLogging;
-					this.persistLogMessages();
-				});
+			this.store.pipe(select(selectStarkLogging)).subscribe((starkLogging: StarkLogging) => {
```

Comment:
Just a suggestion, could we maybe use starkLoggingSelector instead of selectStarkLogging ? :)
It's weird for me having select(select<>). I would prefer if we had something as select(<>Selector)...

Reply:
Mmm I see, well it depends on which point of view you look at it, in my case I thought of this:

1. normally the selector functions return (select) something from the state and as such, the name of the function is imperative: "selectXXXX"
2. follow the guidelines defined in the NGRX-Store API, since these functions will be exposed, keeping in mind that now any developer can use our framework and we should try not to diverge too much (naming conventions, API, ...) from the rest of the ecosystem. Otherwise my fear is that the learning curve of our framework, which is already important, will deter the devs from adopting it